### PR TITLE
[typo] fix typo

### PR DESCRIPTION
**Pull request by zhimin-z**

### CODE REVIEW

**Comment by erbanku** [comment]
Typo Police 👮‍♂️

**Review by heygsc — APPROVED** [review]

**Review by heygsc — APPROVED** [review]

**Review by 0x5459 — APPROVED** [review]
LGTM

**Review by Dirreke — APPROVED** [review]
Nice catch!

**Review by F001 — APPROVED** [review]

**Comment by elin-croft** [comment]
lgtm

**Comment by erbanku** [comment]
LGTM

**Review by erbanku — APPROVED** [review]

**Review by yurhett — APPROVED** [review]

**Review by HuanzhiMao — APPROVED** [review]

**Comment by Cai2w** [comment]
LGTM

**Review by Yuliang-Lee — APPROVED** [review]
LGTM

**Review by thulishuang — COMMENTED** [review]
LGTM

**Review by CoderBak — APPROVED** [review]

**Review by bingzhao-niu — APPROVED** [review]

**Review by Cai2w — APPROVED** [review]

**Review by tengwang0318 — APPROVED** [review]

**Comment by liyiming-autra** [comment]
LGTM

**Review by Linchen-Xu — APPROVED** [review]

**Review by XmchxUp — APPROVED** [review]

**Comment by Zoushen6** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -262,7 +262,7 @@
 （四）**未参加研究或创作而在研究成果、学术论文上署名**，**未经他人许可而不当使用他人署名**，**虚构合作者共同署名**，或者多人共同完成研究而在成果中未注明他人工作、贡献；<br />
 # 总结
 总的来说，我一共有八篇一作SCI论文，其中5篇的实验结果使用了篡改数据，2篇直接抄袭，翻译别人的论文。轻轻松松获得如此丰硕的成果，让我能够包揽从学术新秀，特等奖学金到青拔人才的所有奖项，而老老实实的你，又拿什么来和我比呢？<br />
-通过以上讲解，我相信大家一定受益匪浅。目前中国国内学术风气确实不太好，所以像我这样对学术没有敬畏之心的人才能如鱼得水，急功近利，**伪造科研数据**，将他人的研究成果改头换面据为己有，甚至直接是拿来主义，我认为这种行为是完全没问题的。大部分人可能难以接受我的观点，所以你们只能在科研圈的最底层挣扎，无法像我一样年少有为。尽管在我的行径，不仅无法推进学术发展，还会促使学术氛围的进一步恶化。但这和我有什么关系呢？<br />
+通过以上讲解，我相信大家一定受益匪浅。目前中国国内学术风气确实不太好，所以像我这样对学术没有敬畏之心的人才能如鱼得水，急功近利，**伪造科研数据**，将他人的研究成果改头换面据为己有，甚至直接是拿来主义，我认为这种行为是完全没问题的。大部分人可能难以接受我的观点，所以你们只能在科研圈的最底层挣扎，无法像我一样年少有为。尽管我的行径不仅无法推进学术发展，还会促使学术氛围的进一步恶化。但这和我有什么关系呢？<br />
```
Comment (oldthreefeng) [inline README.md:265]: LGTM

**Review by Zoushen6 — APPROVED** [review]

**Review by leantli — APPROVED** [review]

**Review by ALEX-YRDM — APPROVED** [review]

**Comment by stupidZZ** [comment]
LGTM

File: README.md (hunk 1 above)
Comment (KNKNKNL) [reply README.md:265]: LGTM

**Review by KNKNKNL — APPROVED** [review]

**Review by fishtailstudio — APPROVED** [review]
good job

**Review by Zhang-Zhaoji — APPROVED** [review]

**Review by AnkorTn — APPROVED** [review]
语言大师！

**Review by Orekiz — APPROVED** [review]

**Pull request closed without merging**